spaces/domains.json

```
@@ -216,5 +216,6 @@
   "voting.smartdex.finance": "smartdex-voting.eth",
   "vote.moonpirate.finance": "moonpirate.eth",
   "vote.mai.finance" : "qidao.eth",
-  "dao.modefi.io" : "mod.eth"
+  "dao.modefi.io" : "mod.eth",
+  "vote.conqueruranus.eth":"conqueruranus.eth"
```

A space is missing. Please update the file indentation.
```suggestion
  "vote.conqueruranus.eth": "conqueruranus.eth"
```